### PR TITLE
SWPWA-1390 - Footer redesign

### PR DESCRIPTION
**Pull request by robertsbriedis**
#1390

### CODE REVIEW

**Review by yeegor — CHANGES_REQUESTED** [review]
Minor changes required, mostly readability improvements, although some are related to the logic.

File: src/app/component/Footer/Footer.component.js [hunk 1]
```diff
@@ -43,27 +106,33 @@ export class Footer extends PureComponent {
...
+                </span>
+            </ContentWrapper>
+        );
+    }
+
     render() {
-        const { copyright, isVisibleOnMobile, device } = this.props;
+        const { isVisibleOnMobile, device } = this.props;
 
         if (!isVisibleOnMobile && device.isMobile) {
```
[the remark this answers is not in the document]
Comment (robertsbriedis) [reply src/app/component/Footer/Footer.component.js:137]: One checks if its mobile other if we want to show footer on mobile.

File: src/app/component/Footer/Footer.component.js [hunk 2]
```diff
@@ -35,6 +40,64 @@ export class Footer extends PureComponent {
...
+            <Link
+              block="Footer"
+              elem="ColumnItem"
+              to={ href }
+              mods={ mods }
+            >
+                { this.renderColumnItemContent(src, title) }
+            </Link>
+        );
+    };
```
[the remark this answers is not in the document]
Comment (robertsbriedis) [reply src/app/component/Footer/Footer.component.js:69]: Not quite sure, how I could rename it better.

File: src/app/component/Footer/Footer.config.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                href: '/',
+                title: 'Women'
+            },
+            {
+                href: '/',
+                title: 'Men'
+            },
+            {
+                href: '/',
+                title: 'Accessories'
```
[the remark this answers is not in the document]
Comment (yeegor) [reply src/app/component/Footer/Footer.config.js:56]: Still relevant!

File: src/app/util/Media/Media.js [hunk 4]
```diff
@@ -15,7 +15,11 @@ export const CATEGORY_MEDIA = 'catalog/category/';
...
-export default (src, subPath = '') => {
-    const { ConfigReducer: { secure_base_media_url } } = getStore().getState();
-    return `${ secure_base_media_url || '/media/' }${ subPath }${ src }`;
+export default (src, subPath = '', isMediaPath = true) => {
+    const { ConfigReducer: { secure_base_media_url, base_url } } = getStore().getState();
+    const baseUrl = isMediaPath
+        ? secure_base_media_url || '/media/'
+        : base_url;
+
+    return `${ baseUrl }${ subPath }${ src }`;
```
Comment (yeegor) [inline src/app/util/Media/Media.js:24]: Please provide comments on this

**Review by yeegor — CHANGES_REQUESTED** [review]
Minor changes requested.

**Review by alfredsgenkins — APPROVED** [review]
I think this approach with CMS block or footer styles is interesting, while in real life providing same styles as in your styled footer would be very hard.

**Pull request merged**